Makefile

```diff
@@ -70,25 +70,30 @@ clean:  ## Clean all working/temporary files
 
 # DOCKER STUFF
 up: ## Start server using Docker
-	docker compose up --quiet-pull
+	export EXPLORER_VERSION=$$(uvx setuptools-scm) && \
```

Comment:
I think we should have something like this earlier in the Makefile:
```Makefile
ifdef NO_UV
VERSION_CMD = git describe --tags | sed 's#\-g#+g#'
else
VERSION_CMD = uvx setuptools-scm
endif
```
and then use `$(VERSION_CMD)` inside the parenthesis in the places that now have `$$(uvx setuptools-scm)`. This would make it possible to build the image even for someone who doesn't have uv installed on their machine (me, as one example).